gdal/doc/source/drivers/raster/fits.rst

```diff
@@ -24,10 +25,12 @@ the GDAL handle is closed.
 
 Note to those familiar with the CFITSIO library: The automatic rescaling
 of data values, triggered by the presence of the BSCALE and BZERO header
-keywords in a FITS file, is disabled in GDAL. Those header keywords are
+keywords in a FITS file, is disabled in GDAL < v3.0. Those header keywords are
 accessible and updatable via dataset metadata, in the same was as any
 other header keywords, but they do not affect reading/writing of data
-values from/to the file.
+values from/to the file. Starting from version 3.0 BZERO and BSCALE keywords
+are managed via standard Set/GetOffset() and Set/GetScale GDAL functions and no more
```

Comment:
Please, could you use instead these:

```
:cpp:func:`GDALRasterBand::GetOffset` / :cpp:func:`GDALRasterBand::SetOffset`
```
and
```
:cpp:func:`GDALRasterBand::GetScale` / :cpp:func:`GDALRasterBand::SetScale`
```

Reply:
These directive will autolink the functions from the API reference.

Reply:
Done in https://github.com/OSGeo/gdal/pull/1598/commits/128d8aa5478d62035ec83202c63056e5185a6d7f. Thanks!

Reply:
Thank you